tsconfig.json

```diff
@@ -7,18 +7,20 @@
     "module": "ES6",
     "moduleResolution": "node",
     "paths": {
-      "@/*": ["src/*"]
+      // "@/*": ["src/*"]
```

Comment:
if not necessary, delete. if necessary add a comment why it is commented out

Reply:
I'll add additional comment on this.